### PR TITLE
CSI-2896 more controller renaming

### PR DESCRIPTION
**Pull request by oriyarde**
continuation of https://github.com/IBM/ibm-block-csi-driver/pull/307

### CODE REVIEW

File: controller/controller_server/csi_controller_server.py [hunk 1]
```diff
@@ -104,21 +104,22 @@ def CreateVolume(self, request, context):
                         return build_error_response(message, context, grpc.StatusCode.ALREADY_EXISTS,
                                                     csi_pb2.CreateVolumeResponse)
 
-                    copy_source_res = self._handle_existing_volume_source(volume, source_id, source_type,
-                                                                          array_connection_info.system_id,
-                                                                          context)
-                    if copy_source_res:
-                        return copy_source_res
+                    response = self._handle_existing_volume_source(volume, source_id, source_type,
```
Comment (ArbelNathan) [inline controller/controller_server/csi_controller_server.py:107]: what does `"response"` means?

Comment (ArbelNathan) [reply controller/controller_server/csi_controller_server.py:107]: I understand `"response"` from spec method, not else..

Comment (oriyarde) [reply controller/controller_server/csi_controller_server.py:107]: what else could it be?

Comment (ArbelNathan) [reply controller/controller_server/csi_controller_server.py:107]: could it be more specific? like `create_volume_response`?
if it's not a `spec` method (or something like `utils.generate_csi_create_volume_response` which is clear from the name what the response is.) `response` could be anything..

Comment (oriyarde) [reply controller/controller_server/csi_controller_server.py:107]: would it be clear enough if instead I rename `_handle_existing_volume_source` to something like `_get_response_for_existing_volume_source`?
I'm still not sure what `response` could it be besides `spec`

Comment (ArbelNathan) [reply controller/controller_server/csi_controller_server.py:107]: the `response` is a `grpc` object that of "type" `CreateVolumeResponse`, a `response` could be of any kind and the method name is not telling which one. that why I suggested `create_volume_response` for the name here, every other place indicates the `response` type by the function name. I don't think that `_get_response_for_existing_volume_source` is better for that reason.

Comment (oriyarde) [reply controller/controller_server/csi_controller_server.py:107]: >a response could be of any kind

e.g.?

>the method name is not telling which one

this code is in a method called `CreateVolume`


how about `_get_create_volume_response_for_existing_volume_source`?

Comment (ArbelNathan) [reply controller/controller_server/csi_controller_server.py:107]: > > a response could be of any kind
> 
> e.g.?
`OK`
why? IDK, but why would it be `CreateVolumeResponse`? where is it saying that it is? it's in create volume but it does not create volume..

`_get_create_volume_response_for_existing_volume_source` is ok but long.. if you don't care so Im ok

Comment (oriyarde) [reply controller/controller_server/csi_controller_server.py]: done

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**